disdrodb/data_transfer/scripts/transfer_disdrodb_archive.py

```diff
@@ -0,0 +1,109 @@
+# -----------------------------------------------------------------------------.
+# Copyright (c) 2021-2022 DISDRODB developers
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+# -----------------------------------------------------------------------------.
+####################################################################
+## Wrapper to download disdrodb archives by command lines ##
+####################################################################
+import click
+
+
+def click_upload_download_option(function: object):

```

Comment:
This goes in `download_data.py` and must is imported here.  
The doc is only for download, so call it `click_download_options at this point and add also `--force`.

In the other script interfaces, we use:
`@click.argument("disdrodb_dir", metavar="<disdrodb_dir>")`  so I guess `disdrodb_dir` is only positional argument.

By specifying here with`  "--disdrodb_dir",  required=True` is also positional or the user must specify in the terminal `--disdrodb_dir <path>`?

Side note: I was thinking to prepare a PR where the user can define a `disdrodb_settings.yml` file (to be saved in /home) where the `disdrodb_dir` path can be saved to avoid always replicating it ... so that is not provided, the default is taken (if specified ... otherwise error is rased). At this point  "--disdrodb_dir"  would become optional too ....

Reply:
Hi @ghiggi `click_upload_download_option()` could be used for both upload and download.  `click_download_option()` is specific to the download. That why I put these two functions here and I renamed the file. what do you think ?

I aggree with you for `--disdrodb_dir` I'll chamge that.

Reply:
ok

Reply:
Ok I put the click arguments definition to `disdrodb\data_transfer\download_data.py`  and change the help desc to match only download.